tests/installation/partitioning.pm

```diff
@@ -29,6 +29,7 @@ sub run {
         $cmd{addraid}         = 'alt-i';
         $cmd{filesystem}      = 'alt-a';
         $cmd{exp_part_finish} = 'alt-n';
+        $cmd{size_hotkey}     = 'alt-s';
```

Comment:
Maybe I am wrong, but "Define changed shortcuts" comment - is this needed here if susedistribution.pm has the same shortcut?

Reply:
where did you spot this in susedistribution?

Reply:
This PR is adding 2 lines of code - I am wondering if first line is not enough

Reply:
ah now I got it, @rwx788 will handle accordingly

Reply:
My bad. @kravciak Thanks a lot!